docker/bin/await-service

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+# Wait for a service to accept connections.
```

Comment:
apiserver has /healthz

there's a PR to add health endpoints to other apps. needs review.

On Mon, May 18, 2015 at 5:14 PM, Karl Isenberg <notifications@github.com>
wrote:

> In docker/bin/await-service
> <https://github.com/mesosphere/kubernetes-mesos/pull/282#discussion_r30548038>
> :
>
> > @@ -0,0 +1,30 @@
> > +#!/usr/bin/env bash
> > +
> > +# Wait for a service to accept connections.
>
> Good point. I see Etcd has a /health endpoint for that purpose.
> Do the other components have one we should use too?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mesosphere/kubernetes-mesos/pull/282/files#r30548038>.
>